### PR TITLE
Further polish d2k tileset - continuation

### PR DESCRIPTION
**Pull request by CH4Code**
Depends on/Hijacks #12868.

- implements Items comments on passability on Bridges and some Sand-Detail terrain.
- fixes the map importer for the terrain tiling error Item mentioned.
- renames and reorders terrain categories: To be uniform (XXX-YYY-Cliff), similar categories adjacent to each other, moves most used categories to top.
- moves tiles into appropriate categories

### CODE REVIEW

**Comment by reaperrr** [comment]
> OpenRA.Mods.D2k/UtilityCommands/D2kMapImporter.cs:L413: [SA1508] A closing curly bracket must not be preceded by a blank line.

**Comment by CH4Code** [comment]
I went trough Sand-Detail and Ice-Detail comparing with my eyes and mind and found 21 true or very close duplicates, 14 composite tiles, and 16 new smallest tiles. See image or [map](http://resource.openra.net/maps/21273/). 

Composite tiles would be tiles that can be split into smaller parts without beeing noticeable. Those can be removed and replaced by smaller tiles, which are indivisible.
This means there could be 21+14 -16 = 19  less tiles in the Sand-Detail and Ice-Detail categories.

I have a list here (corresponding to the map/picture), which tells me which (lowest ID) tile a composite part or a duplicate tile duplicates. I need that later for the UpgradeRules.cs where old maps using those redundant tiles get the new tile replacements.


duplicates_sand

File: mods/d2k/tilesets/arrakis.yaml [hunk 1]
```diff
@@ -4240,140 +4240,140 @@ Templates:
...
 		Images: BLOXXMAS.R8
 		Frames: 350, 351, 352, 370, 371, 372, 390, 391, 392
 		Size: 3,3
-		Category: Rock-Detail
+		Category: Bridge
 		Tiles:
 			0: Rock
 			1: Rock
 			2: Rock
 			3: Cliff
```
Comment (ltem) [inline mods/d2k/tilesets/arrakis.yaml]: `Tile 0` and `Tile 2` should be changed to `Cliff` 

image

**Review by ltem — COMMENTED** [review]
one small issue found

**Comment by ltem** [comment]
:+1:

**Comment by reaperrr** [comment]
@MustaphaTR Would you mind taking a look at this?

**Comment by MustaphaTR** [comment]
329, 342, 343 and 364 doesn't look like belong to Rock-Detail. 329, 342, and 364 can be moved to Sand-Rock-Cliff, 343 can go to either Sand-Rock-Cliff or Bridge. It would be cool if that can be addressed in this PR too.

The rest looks fine to me.

**Comment by CH4Code** [comment]
Rebased

**Comment by CH4Code** [comment]
I also squashed  a lot of the commits, so this looks nicer in the commit history when merged.

**Comment by CH4Code** [comment]
@MustaphaTR. I did your proposed changes.

In addition to that I moved some more tiles in wrong categories (from `sand-detail` and `ice-detail`   to `sand-detail`, `ice-detail`, `rotten-base`) to the more appropriate ones.
Basically all rocks in the destert are now in `sand-detail`. 
All river beds, ice and ponds in `ice-detail`.
All tracks and smudges of former humaniod influence are now in `rotten-base`.

This should solve ("x is in wrong category") once and for all ^^

**Review by MustaphaTR — APPROVED** [review]

**Review by reaperrr — APPROVED** [review]

**Comment by reaperrr** [comment]
https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/e7618f5d466ed33496075d60ed9f35e82e37e9af...1f54a940ad31b3b62e057dc56660808a492ac003

**Comment by CH4Code** [comment]
nice :)

**Pull request merged**